meson.build

```diff
@@ -102,7 +102,7 @@ WITH_WAYLAND = false
 if get_option('with_wayland') != 'no'
   wayland_dep = dependency('wayland-client', version : '>= 1.11.0',
 			   required : get_option('with_wayland') == 'yes')
-  wayland_scanner_dep = dependency('wayland-scanner',
+  wayland_scanner_dep = dependency('wayland-scanner', version : '>= 1.15',
```

Comment:
From commit message:
```
The "code" option was deprecated by yours truly with wayland-scanner 1.15.
```
I don't understand this sentence, can you, please, reformulate?

Reply:
before wayland-scanner 1.15 there was not `code-private` only `code`

Reply:
Aside: @dvrogozh is there a IRC/Slack/other channel that you're on - could not see you on the official ones.

Reply:
>  "by yours truly" as bringing confusion

right:). I really would appreciate simpler commit message.

> @dvrogozh is there a IRC/Slack/other channel that you're on - could not see you on the official ones.

I afraid I am not using those.